plugins/plugin-chart-echarts/src/Timeseries/transformProps.ts

```diff
@@ -169,12 +170,16 @@ export default function transformProps(chartProps: ChartProps): EchartsProps {
     },
     tooltip: {
       ...defaultTooltip,
-      trigger: 'axis',
-      formatter: params => {
+      trigger: richTooltip ? 'axis' : 'item',
+      formatter: (params: any) => {
+        const value: any = !richTooltip ? params.value : params[0].value[0];
+        const prophetValue: any = !richTooltip ? [params] : params;
+
         // @ts-ignore
-        const rows = [`${smartDateVerboseFormatter(params[0].value[0])}`];
+        const rows: any = [`${smartDateVerboseFormatter(value)}`];
         // @ts-ignore
-        const prophetValues = extractProphetValuesFromTooltipParams(params);
+        const prophetValues: any = extractProphetValuesFromTooltipParams(prophetValue);
```

Comment:
As @geido pointed out, it would be nice to get rid of any explicit any, and avoid ts-ignore.

If we MUST use the `any` could we specifically ignore the `no-explicit-any` rule rather than completely `ts-ignore`-ing the entire line?